ppo/train.py

```diff
@@ -175,7 +173,7 @@ def __init__(
         self.embedding = nn.ModuleDict(
             {
                 name: nn.Sequential(
-                    InputNorm(len(entity.features)),
+                    InputNorm(len(entity.features), cliprange=1),
```

Comment:
I think this is too aggressive, it will clip 32% of normally distributed values which could throw away a lot of information (and could clip even much more for non-normally distributed input features). The original default of 5 wasn't chosen in a super principled way (I just cargo-culted it from OpenAI Five), so there might be a better value, but it still seems about right.

Reply:
Yes this is something I just wanted to test, this should be removed, my bad.

Reply:
E.g. 42% of uniformly distributed values would get clipped, and this would be even worse for more skewed distributions:
```
>>> import numpy as np
>>> np.random.uniform(-1.73, 1.73, size=(100000,)).std()
0.9988141429161412
>>> (0.73 * 2) / (1.73 * 2)
0.42196531791907516
```